### ISSUE TITLE
Non-ASCII strings not handled properly

### ISSUE THREAD

**Issue by vincentisambart** [joeferner/node-bplist-creator#11]
The binary plist file generated by:
```js
var bplist = require('bplist-creator');
var buffer = bplist({
  key1: "テスト"
});
```
gives a plist file with key1's value being garbage (`ãã¹ã`) when opened in Xcode's plist editor.

When forcing the string type to "string-utf16" like I do below it works fine though:
```js
var bplist = require('bplist-creator');
var buffer = bplist({
  key1: {
    bplistOverride: true,
    value: "テスト",
    type: "string-utf16"
  }
});
```

Looking at your code, you output a type 6 for UTF-16 and type 5 for UTF-8, but looking at Core Foundation's binary plist handling [source code](http://opensource.apple.com/source/CF/CF-855.17/ForFoundationOnly.h), there's no UTF-8 string format, only:
```c
    kCFBinaryPlistMarkerASCIIString = 0x50,
    kCFBinaryPlistMarkerUnicode16String = 0x60,
```
so it seems that 5 is for ASCII, not UTF-8.

You should probably use the `entry.type === 'string-utf16'` path if the string is not ASCII-only.

**Comment by vincentisambart** [comment]
Oops, it seems it should be fixed by c68cc09e3bfec7d619bf0b55a5294f9a5fee6f01.
Then it would be nice to have a new version released. Sorry for the trouble.

**Comment by joeferner** [comment]
Just published v0.0.5

**Comment by vincentisambart** [comment]
Thank you!
